### DISCUSSION TITLE
how best to change user Group structure

### DISCUSSION THREAD

**Discussion by buddycorb** [thoughtspot/cs_tools#187] [Q&A]
I'm changing my user Groups to a new pattern. Wondering about the best way to re-align content with group permissions as I do so (most groups are similar to their predecessor). I see user-management module's 'rename' feature would be good, but that appears to be users only not groups. I've used Searchable in the past to get list of objects and permissions, etc., so I have a guide to how things have been shared, but that'd still be a manual editing process. Any thoughts? thank you!

**Comment by boonhapus** [comment]
Hey @buddycorb ,

We could potentially do this with `user-management sync`.. could you give a small example of what you're trying to do when you say "reorganize"?

Is this about moving users to new groups? Or changing privilege structure on groups? Or something else altogether?

**Comment by buddycorb** [comment]
Thanks for responding!
We're migrating to cloud, and will be going from LDAP to SAML a.d. groups. The SAML ones are new, though similar to my old LDAP ones.
So, for an old LDAP group like 'Thoughtspot Operations Users' which might have 50 or so people in it, that group has permissions to various tables, worksheets/models, answers, and liveboards.
Now I'll have a new SAML group like 'Workday - Operations', which might have 150 or so people some of whom might be current users in Thoughtspot (some of those 50), and others may not be, though I don't mind having their user profile come in - goal is to onboard everyone eventually anyway.

I want to give the new SAML group the same content permissions as the old LDAP group, then delete the LDAP group.
I would then want to do the same process for about 15 other groups.
